ci/licenses_golden/licenses_third_party

```diff
@@ -27249,6 +27251,95 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ====================================================================================================
 
+====================================================================================================
+LIBRARY: tinygltf
+ORIGIN: ../../../third_party/tinygltf/LICENSE
+TYPE: LicenseType.mit
+FILE: ../../../third_party/tinygltf/appveyor.yml
+FILE: ../../../third_party/tinygltf/loader_example.cc
+FILE: ../../../third_party/tinygltf/premake5.lua
+FILE: ../../../third_party/tinygltf/stb_image.h
+FILE: ../../../third_party/tinygltf/stb_image_write.h
+FILE: ../../../third_party/tinygltf/tiny_gltf.cc
+----------------------------------------------------------------------------------------------------
+MIT License
+
+Copyright (c) 2017 Syoyo Fujita, Aurélien Chatelain and many contributors
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+====================================================================================================
+
+====================================================================================================
+LIBRARY: tinygltf
+ORIGIN: ../../../third_party/tinygltf/json.hpp
```

Comment:
We already have a copy of this JSON dependency. Lets use that instead and ignore this one.

Reply:
Looks like it's not a top level dependency -- the compiler is getting it from `//third_party/inja/third_party/include`. Untangling this at the moment.

Reply:
Added `nlohmann/json` config to the buildroot PR and got rid of the 3rd party includes in inja. Should be good to go once I fix up the licenses.